### DISCUSSION TITLE
Does Tabby supports keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by vkpdeveloper** [Eugeny/tabby#5740] [Q&A]
Dear amazing people, I loved this and now it's my default terminal. I am gonna use it everywhere.

I have one important thing to ask does it support keyboard shortcuts because everything is just amazing but no keyboard shortcuts mean no real fun.

Let me know about it.

Thank you guys for this amazing contribution.

**Comment by Eugeny** [comment]
If you mean macros/quick commands, take a look at the "quick-cmds" plugin.
